collector/components_test.go

```diff
@@ -0,0 +1,40 @@
+// Code generated by "go.opentelemetry.io/collector/cmd/builder". DO NOT EDIT.
+
+package collector
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/component/componenttest"
+)
+
+func TestValidateConfigs(t *testing.T) {
+	factories, err := Components()
+	assert.NoError(t, err)
```

Comment:
Interesting -- I thought the `assert` library approach wasn't recommended in Go. Can we avoid this to keep this compliant with the general style guide?

Reply:
Yeah I agree - but this is generated code so I figured I would stay coherent. 

Anyway, I'll change it to non-assert.

Reply:
Oh, the unit tests are autogenerated?

Reply:
The component ones are, the others aren't

Reply:
I see -- yes, for code that we write as humans please let's avoid assertion libraries.